docs/CONTRIBUTING.md

```diff
@@ -148,6 +157,20 @@ git clone https://github.com/enso-org/enso.git
 git clone git@github.com:enso-org/enso.git
 ```
 
+### Getting Set Up
```

Comment:
Please mention that it only concerns the new, completely unused, probably-not-what-you're-here-for parser. Particularly that if I understand correctly, it's going to be automated when the parser is actually needed.

Reply:
Plan is to add the initial bootstrap steps for the Java 11 stuff here as well once radek's PR merges. What do you think?

Reply:
Should be separate sections, one for SBT strangeness, one for "if you want to do rust". Merge after rust is in SBT.